src/bionics.cpp

```diff
@@ -2692,10 +2692,28 @@ int Character::get_free_bionics_slots( const bodypart_id &bp ) const
     return get_total_bionics_slots( bp ) - get_used_bionics_slots( bp );
 }
 
-bool Character::cbm_needs_anesthesia() const
+static bool cbm_needs_anesthesia( Character &who )
```

Comment:
```suggestion
auto cbm_needs_anesthesia( const Character &who ) -> bool
```

`static` makes function local to file which means it cannot be used outside bionics.cpp: https://en.cppreference.com/w/cpp/language/storage_duration#Storage_duration